sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -75,6 +75,7 @@ abstract class Expression extends TreeNode[Expression] {
    * - it relies on some mutable internal state, or
    * - it relies on some implicit input that is not part of the children expression list.
    * - it has non-deterministic child or children.
+   * - it is an UDF that can cause runtime exception on some specific input.
```

Comment:
how about `it assumes the input satisfies some certain condition via the child operator`?

Reply:
Ok.